packages/metal-quartz-link/src/MetalQuartzLink.js

```diff
@@ -0,0 +1,104 @@
+'use strict';
+
+import Component from 'metal-component';
+import MetalQuartzIcon from 'metal-quartz-icon';
+import Soy from 'metal-soy';
+import { core, object } from 'metal';
+import { validators } from 'metal-state';
+
+import templates from './MetalQuartzLink.soy.js';
+
+/**
+ * Implementation of the Metal Quartz Link.
+ */
+class MetalQuartzLink extends Component {
+	/**
+	 * @inheritDoc
+	 */
+	created() {
+		MetalQuartzLink.instances.push(this);
```

Comment:
Lets say you have a component MetalQuartzList with some state called `order` that can be `alphabetical, date, or status`. I add some buttons that lets you change the `order` but I want to use jQuery instead of Metal. I want to be able to:

$('.my-button-alphabetical').on('click', function() {
    dom.getComponent('#myListId').order = 'alphabetical';
});

$('.my-button-date').on('click', function() {
    dom.getComponent('#myListId').order = 'date';
});

This is also useful for two unrelated Metal Components to be able to talk to each other.